packages/orbit-components/src/InputGroup/README.md

```diff
@@ -28,11 +28,11 @@ Table below contains all types of the props available in InputGroup component.
 | error        | `React.Node`                |              | The error to display beneath the InputGroup. [See Functional specs](#functional-specs)                                                                         |
 | disabled     | `boolean`                   |              | Whether to disable all nested fields.                                                                                                                          |
 | flex         | `string` or `Array<string>` | `"0 1 auto"` | The flex attribute(s) for children of the InputGroup. [See Functional specs](#functional-specs)                                                                |
-| help         | `React.Node`                |              | The help to display beneath the InputGroup.                                                                                                                    |
 | label        | `Translation`               |              | The label for the InputGroup. [See Functional specs](#functional-specs)                                                                                        |
 | onChange     | `event => void \| Promise`  |              | Function for handling onClick event. [See Functional specs](#functional-specs)                                                                                 |
 | onFocus      | `event => void \| Promise`  |              | Function for handling onFocus event. [See Functional specs](#functional-specs)                                                                                 |
-| onBlur       | `event => void \| Promise`  |              | Function for handling onBlur event. [See Functional specs](#functional-specs)                                                                                  |
+| onBlur       | `event => void \| Promise`  |              | Function for handling onBlur event between different InputGroup children. [See Functional specs](#functional-specs)                                            |
+| onBlurGroup  | `event => void \| Promise`  |              | Function for handling onBlur event for the whole InputGroup. [See Functional specs](#functional-specs)                                                         |
```

Comment:
This is not clear enough, but the full explanation would be too long for this table, so I propose adding it to functional specs instead (the short description directs there anyway), below the point about event handlers:

> - In comparison to `onBlur`, which is triggered by every blur event of InputGroup's children, `onBlurGroup` treats children as a single field, and fires only when a child loses focus and no child gains focus, for example clicking out of InputGroup.